### PR TITLE
[PLAY-25] Improve number spacing global props docs 

### PR DESCRIPTION
**Pull request by GordonHodanich**
#### Screens

Screen Shot 2021-11-17 at 10 31 56 AM

#### Breaking Changes

No, this fix added a props/values section as well as a code snippet.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/PLAY-25

#### Checklist:

- [x] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [x] **SCREENSHOT** Please add a screen shot or two.
- [x] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

### CODE REVIEW

**Comment by jasoncypret** [comment]
Great Job!

I decided to make a quick design to nail the feedback. Just a couple of small things

Spacing


1. Let's add some microscopy, titles, and captions
2. Bring the example inside the card + use a table to highlight the difference
3. Simplify the code example to just be the component but full react setup
4. SHIPIT! 🚀

**Comment by kyleschechter** [comment]
Wonderful!

**Comment by GordonHodanich** [comment]
One issue I was unable to resolve was the formatting of the prop "numberspacing". It looks like there is a formatting issue not allowing me to make it camel case @jasoncypret

File: playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_number_spacing.html.slim [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
-        p $1,231,123,123.00
-        p $7,444,112,512.00
-= pb_rails("card", props: { max_width: "sm", number_spacing: "tabular", margin_top: "sm" })
-    code tabular 
-    = pb_rails("body")
-        p $1,231,123,123.00
-        p $7,444,112,512.00
+= pb_rails("card", props: { text: "visual guide", max_width: "xl" })
+    = pb_rails("caption", props: {text: "Visual Guide", margin_bottom: "sm" })
+    = pb_rails("card", props: { max_width: "xl" })
```
Comment (jasoncypret) [inline playbook-website/app/views/pages/visual_guidelines_partials/_pb_doc_number_spacing.html.slim:13]:
```suggestion
```
This is just adding an extra border. Lets remove

**Review by jasoncypret — APPROVED** [review]

**Pull request merged**